Makefile

```diff
@@ -3,7 +3,6 @@ BEAT_PATH=github.com/raboof/connbeat
 SYSTEM_TESTS=true
 TEST_ENVIRONMENT?=true
 ES_BEATS?=./vendor/github.com/elastic/beats
-BEAT_GOPATH=/vagrant/gocode
```

Comment:
Was this necessary?

Reply:
No it was just a trial to check a scenario but realized later it wasn't needed.